corehq/motech/openmrs/views.py

```diff
@@ -35,10 +35,19 @@
 PASSWORD_PLACEHOLDER = '*' * 16
 
 
-class OpenmrsRepeaterView(AddCaseRepeaterView):
+class AddOpenmrsRepeaterView(AddCaseRepeaterView):
     urlname = 'new_openmrs_repeater$'
-    page_title = "Forward to OpenMRS"
-    page_name = "Forward to OpenMRS"
+    page_title = ugettext_lazy("Forward to OpenMRS")
+    page_name = ugettext_lazy("Forward to OpenMRS")
+
+
+class EditOpenmrsRepeaterView(EditRepeaterView, AddOpenmrsRepeaterView):
+    urlname = 'edit_openmrs_repeater'
+    page_title = ugettext_lazy("Edit OpenMRS Repeater")
+
+    @property
+    def page_url(self):
+        return EditRepeaterView.page_url
```

Comment:
How does that work? It would not need an instance of `EditOpenmrsRepeaterView`?

Reply:
Because this resolved an error, I assumed `self` was being passed as if the property was being called with `EditRepeaterView.page_url(self)`. (If you do call it like that, you get `TypeError: 'property' object is not callable`.) ... But I've just tested this in a shell, and it turns out I'm wrong. The view renders and the form works ... but here is what's actually happening: 
```python
In [13]: class Foo(object):
    ...:     @property
    ...:     def bar(self):
    ...:         return str(type(self))
    ...:         
    ...:         

In [14]: class Baz(Foo):
    ...:     @property
    ...:     def bar(self):
    ...:         return Foo.bar
    ...:     

In [15]: baz = Baz()

In [16]: baz.bar
Out[16]: <property at 0x7f8ea7718520>

In [17]: foo = Foo()

In [18]: foo.bar
Out[18]: "<class '__main__.Foo'>"
```

I'll fix it.